apps/bwclk/app.js

```diff
@@ -277,9 +277,9 @@ let drawLock = function() {
 
 let drawWidgets = function() {
   if(isFullscreen()){
-    for (let wd of WIDGETS) {wd.draw=()=>{};wd.area="";}
+    widget_utils.hide();
   } else {
-    Bangle.drawWidgets();
+    Bangle.drawWidgets(); // Should this be replaced with widget_utils.show(); ?
```

Comment:
Should this be replaced with widget_utils.show(); ?

Reply:
Tried this on another clock app, didn't work very well so I'll not change to .show() here.